docs/helm/helm.md

```diff
@@ -21,11 +21,11 @@ Common actions from this point include:
 - helm list:      list releases of charts
 
 Environment:
```

Comment:
I wonder should all the environment variables be put in a bullet point list? At the moment when MD rendered, they are just one big blob of text?